jcl/src/java.base/share/classes/com/ibm/oti/util/ExternalMessages-MasterIndex.properties

```diff
@@ -1282,3 +1282,6 @@ K0703="Negative length"
 K0800="Invalid Unicode code point - {0}"
 K0801="Last character in replacement string can't be \, character to be escaped is required."
 K0802="Last character in replacement string can't be $, group index is required."
+
+#java.lang.ref.Reference
+K0900="Create a new Reference since a Reference cannot be cloned."
```

Comment:
Please put a comma after the first "Reference".

Reply:
fixed.